web-registry/src/components/organisms/EntityDisplayForm.tsx

```diff
@@ -399,15 +406,21 @@ const EntityDisplayForm: React.FC<EntityDisplayFormProps> = ({
                 const report = await validate(
                   graphData.shaclGraphByUri.graph,
                   value,
-                  'regen:ProjectPageEntityDisplayGroup',
+                  'http://regen.network/ProjectPageEntityDisplayGroup',
```

Comment:
this is how the JS `SHACLValidator` works from our fork of `rdf-validate-shacl` https://github.com/regen-network/rdf-validate-shacl
it needs the full uri because it doesn't necessarily know the context associated to the provided group
Although we could potentially improve that and use the context from the `value` but this will require some more work on our forked repo